### ISSUE TITLE
"Delete all alarms" schedules an alarm

### ISSUE THREAD

**Issue by roadriverrail** [MycroftAI/skill-alarm#12]
Easy to repro:  Say "Hey Mycroft, delete all alarms"
Expected result: "You deleted X alarms"
Actual result: "Okay, setting a recurring delete all alarms alarm *alarm noise*"

Looks like a lot of stuff falls into the alarm creation code, since virtually all alarms in the system were the result of ambient conversaion.

**Comment by roadriverrail** [comment]
There's definitely something wrong with the intent parser here.  "Hey Mycroft wake up" caused it to set a recurring alarm called "wake up" and trigger the alarm noise immediately.

**Comment by penrods** [comment]
Should be resolved by #26 once it is merged into the Mycroft Skills repo

**Comment by penrods** [comment]
Should be working now
